.travis.yml

```diff
@@ -46,7 +48,9 @@ install:
  # Travis hangs without -j2 argument to ninja.
  - ninja -j2 -C $BUILD_PATH mock_runtime_test mock_main
  - ninja -j2 -C $BUILD_PATH deno
+ - cd ..
```

Comment:
not necessary but if you want to be in the root, then rather use pushd / popd

Reply:
test failed without this see #292